pywb/templates/search.html

```diff
@@ -10,20 +10,16 @@
 {% endblock %}
 
 {% block body %}
-<div class="container-fluid">
-    <div class="row justify-content-center">
-        <h4 class="display-4">
-            {% trans %}Stanford Web Archive Portal{% endtrans %}
-        </h4>
-    </div>
+<div class="row align-items-center bg-image" id="home_header">
+    <h2 class="display-2">{{ _('Stanford Web Archive Portal') }}</h2>
 </div>
 <div class="container mt-4">
     <form class="needs-validation" id="search-form" novalidate>
         <div class="form-row">
             <div class="col-12">
                 <label for="search-url" class="lead" aria-label="Search For Col">
                     {% set coll_title = metadata.title if metadata and metadata.title else coll %}
-                    {% trans %}Locate Archived Sites by URL:{% endtrans %}
+                    {% trans %}Locate Archived Sites by entering URL:{% endtrans %}
```

Comment:
Trivial, but: should `entering` be capitalized since the rest of the text appears to be in title case?

Reply:
Actually - looking at the mock up, it's the opposite (Archived -> archived), but good catch.